shaders/glsl/banner.fragment

```diff
@@ -39,7 +39,7 @@ void main()
 #endif
 
 #ifdef ENABLE_FOG
-	//apply fog
+	// Apply fog

```

Comment:
Comment most of the times begins with low case letter. There is no need to change it imo. And in all other places too ofc. But offsets are ok

Reply:
There are a lot of cases where you used capitals in your comments though, and I think they stop the comments from looking like they are just part of the code if you aren't using highlighting

Reply:
Ok